scripts/WinInstallFiles/InstallJMRI.nsi

```diff
@@ -483,104 +486,10 @@ SectionGroup "JMRI Core Files" SEC_CORE
     ; -- that might not be consistent with this new version
     RMDir /R "$OUTDIR\classes"
 
-	; -- Delete OBE JSON processing files as of JMRI 4.15.5
-    Delete "$OUTDIR\lib\jackson-annotations-2.8.11.jar"
-    Delete "$OUTDIR\lib\jackson-core-2.8.11.jar"
-    Delete "$OUTDIR\lib\jackson-databind-2.8.11.jar"
-    Delete "$OUTDIR\lib\json-schema-validator-0.1.19.jar"
-	
-    ; -- Delete insecure jackson jar files as of JMRI 4.11.3
-    Delete "$OUTDIR\lib\jackson-annotations-2.8.5.jar"
-    Delete "$OUTDIR\lib\jackson-core-2.8.5.jar"
-    Delete "$OUTDIR\lib\jackson-databind-2.8.5.jar"
-
-    ; -- Delete old PJC file for JMRI 4.7.5
-    Delete "$OUTDIR\lib\jna-4.2.2.jar"
-
-    ; -- Delete old jackson jar files as of JMRI 4.7.1
-    Delete "$OUTDIR\lib\jackson-annotations-2.0.6.jar"
-    Delete "$OUTDIR\lib\jackson-core-2.0.6.jar"
-    Delete "$OUTDIR\lib\jackson-databind-2.0.6.jar"
-
-    ; -- Delete old vecmath.jar files as of JMRI 4.5.1
-    Delete "$OUTDIR\lib\vecmath.jar"
-
-    ; -- Delete old USB library files
-    Delete "$OUTDIR\ch.ntb.usb.jar"
-    Delete "$OUTDIR\lib\ch.ntb.usb.jar"
-
-    ; -- Delete old .jar & support files in destination directory
```

Comment:
These three files should still be included for explicit removal - seem to have missed being transferred to the section around https://github.com/JMRI/JMRI/pull/7432/files#diff-735059f12e8148114622f09cfca2fea4R542

Reply:
For clarity, the lines I'm on about are:

```
    Delete "$OUTDIR\jh.1.1.2.jar"
    Delete "$OUTDIR\jh.jar"
    Delete "$OUTDIR\jdom-jdk11.jar"
```

Reply:
Would it be preferable to use `Delete "$OUTDIR\*.jar` to remove all JARs in the JMRI installation root?

Reply:
I've made the requested correction. I think a later PR could address my question.